Src/java/tools/cql-formatter/src/test/resources/org/cqframework/cql/tools/formatter/git-issue-613.cql

```diff
@@ -0,0 +1,5 @@
+define "Test":
```

Comment:
Please add a comment to the beginning of the library with a link to the github issue and a short description explaining the bug and the resolution